crd-convert/src/main/java/io/strimzi/kafka/crd/convert/converter/KafkaConverter.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.crd.convert.converter;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeCreator;
+import io.strimzi.api.annotations.ApiVersion;
+import io.strimzi.api.kafka.model.Kafka;
+import io.strimzi.api.kafka.model.listener.arraylistener.ArrayOrObjectKafkaListeners;
+
+import static java.util.Arrays.asList;
+
+public class KafkaConverter extends Converter<Kafka> {
```

Comment:
We're going to need converts for the other CR kinds as well. I think the changes for tolerations and affinity were done systematically for `KafkaConnect`, `KafkaConnectS2I` and possibly others. There will be further changes for the external logging config stuff that @sknot-rh is working on, plus anything else we identify.

I assume we're also going to add beta versions for those which are current alpha (E.g. the rebalance resource). Right @ppatierno @scholzj ?

Reply:
Do we have this changes documented somewhere?
Or how can I know what all needs to be converted?